libs/sysplugins/smartycompilerexception.php

```diff
@@ -18,9 +18,9 @@ public function __toString()
     /**
      * The line number of the template error
      *
-     * @type int|null
+     * @type int
      */
-    public $line = null;
+    public $line;
```

Comment:
Unfortunately, this does not solve the issue for us. We are still getting the following error:

```NOTICE: PHP message: PHP Fatal error:  Type of SmartyCompilerException::$line must be int (as in class Exception) in /var/ftproot/www/symfony/lib/vendor/smarty/smarty/libs/sysplugins/smartycompilerexception.php on line 8```

If I change the line to `public int $line;`, everything is working, but this is not backward compatible to php < 7.4